bowtie/_cli.py

```diff
@@ -520,14 +524,24 @@ def _validation_results_table(
 
         for test, test_result in test_results:
             subtable.add_row(
-                Text(json.dumps(test.instance)),
-                *(
-                    Text(test_result[each.id].description)
-                    for each in implementations
+                Syntax(
+                    json.dumps(test.instance),
+                    lexer=JSON_SCHEMA_LEXER,
```

Comment:
This isn't right, is it? The instance isn't a JSON Schema, it should use the normal JSON lexer.